adagrams/game.py

```diff
@@ -1,11 +1,156 @@
+import random 
+"""
+Define a list of tuples, to access frequency of letter and letter at same index
+"""
+LETTER_POOL = [
+    ("A", 9),
+    ("B", 2),
+    ("C", 2),
+    ("D", 4),
+    ("E", 12),
+    ("F", 2),
+    ("G", 3),
+    ("H", 2),
+    ("I", 9),
+    ("J", 1),
+    ("K", 1),
+    ("L", 4),
+    ("M", 2),
+    ("N", 6),
+    ("O", 8),
+    ("P", 2),
+    ("Q", 1),
+    ("R", 6),
+    ("S", 4),
+    ("T", 6),
+    ("U", 4),
+    ("V", 2),
+    ("W", 2),
+    ("X", 1),
+    ("Y", 2),
+    ("Z", 1)
+]
+
 def draw_letters():
-    pass
+    """
+    Create a list number_of_letter_available of same lenght as LETTER_POOL
+    Generate random integer, to get random index to access number_of_letters_available
+    If number is greater than one, add to letter_bank list and subtract one, at the given index
+    Returns letter_bank
+    """
+    letter_bank = []
+    number_of_letter_available = []
+    
+    for letter in LETTER_POOL:
+        number_of_letter_available.append(letter[1])
+    
+    while len(letter_bank) < 10:    
+        random_index = random.randint(0, len(LETTER_POOL) - 1)
+        if number_of_letter_available[random_index] > 0:
+            letter_bank.append(LETTER_POOL[random_index][0])
+            number_of_letter_available[random_index] -= 1
+ 
+    return letter_bank
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    """
+    Create a dictionary looping over letter in letter_bank 
+    Create key using letter, sum quantities of given letter key
+    Create dictionary for letter in word, similarly 
+    Returns True or False
+    """
```

Comment:
To reinforce a comment I made above, this is a great description of the function. But as a thought, in production-level code, I would either want this description to be more conceptual or add context (for example "Returns True or False if the given word uses available letters in the given letter_bank"), or for these specific comments to be moved closer to the lines they describe, or deleted